content/en/apps/guides/hosting/ec2-setup-guide.md

```diff
@@ -1,110 +1,123 @@
-- Understanding the log and what the entries mean
+1. Configure couch2pg
+    See the [couch2pg basic configuration](https://github.com/medic/medic-couch2pg/blob/master/README.md) in the `medic-couch2pg` repository.
 
-## Troubleshooting
-#### 1. Restarting processes
-- [How to access container, retrieve logs, isolate security groups]({{< ref "apps/guides/hosting/self-hosting#how-to-access-container-retrieve-logs-isolate-security-groups" >}})
-- [MedicOS service management scripts](https://github.com/medic/medic-os#user-content-service-management-scripts)
+1. Setup postgres to work with couch2pg
+    - Creating the database, setting up permissions, exploring the tables and what they store
 
-#### 2. Investigating logs
-- [Log into container](https://docs.docker.com/engine/reference/commandline/exec/)
-- [Helpful docker commands]({{< ref "core/guides/docker-setup#helpful-docker-commands" >}})
-- Inside container, all appropriate logs can be found in: `/srv/storage/<service_name>/logs/*.log`
+1. Debugging couch2pg/postgres
+    - Understanding the log and what the entries mean
 
-#### 3.Upgrading the container
-- Backup all data (EBS) 
-- Log into container and stop all services
-- DO NOT REMOVE `/srv/storage/medic-core/`, `/srv/settings/medic-core/couchdb/local.ini`, wipe all other files in /srv [Note: Make script publicly accessibile]
-- [Change the image tag to the newest image release version]({{< ref "core/guides/docker-setup#use-docker-compose" >}})
-- [Change image tag in docker-compose file]({{< ref "core/guides/docker-setup#use-docker-compose">}})
-- Launch new containers with appropriate COUCHDB_ADMIN_PASSWORD & HA_PASSWORD environment variables
+## Troubleshooting
 
-#### 4.Upgrading the webapp
-- Use Admin GUI page
-- [CLI via horticulturalist]({{< ref "apps/guides/hosting/self-hosting#links-to-medic-documentation-for-horticulturalist-for-upgrades" >}})
+1. Restarting processes
+    - [How to access container, retrieve logs, restart services]({{< ref "core/guides/docker-setup#helpful-docker-commands" >}})
+    - [MedicOS service management scripts](https://github.com/medic/medic-os#user-content-service-management-scripts)
 
-#### 4a. RDS help
-- [Amazon user guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html)
+1. Investigating logs
+    - [Helpful docker commands]({{< ref "core/guides/docker-setup#helpful-docker-commands" >}}) (includes getting shell on containers)
+    - Inside container, all appropriate logs can be found in: `/srv/storage/<service_name>/logs/*.log`
 
-## Restore from backup and monitoring
-#### 1. Configure backups
-- [EBS Snapshot Lifecycle Manager](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snapshot-lifecycle.html)
+1. Upgrading the container
+    - Backup all data (EBS) 
+    - Log into container and stop all services
+    - DO NOT REMOVE `/srv/storage/medic-core/`, `/srv/settings/medic-core/couchdb/local.ini`, wipe all other files in `/srv` [Note: Make script publicly accessible]
```

Comment:
I think it's a good idea to mention why they shouldn't delete this. We should mention that the data lives in `medic-core` and settings modifications live in `local.ini`. Also, I think it's worth checking that couchdb doesn't save configuration changes in any other settings files. I remember investigating this at some point but don't remember the conclusions I came to.

Reply:
Awesome - good to call out! 

The content you're commenting on is what we [already have there](https://docs.communityhealthtoolkit.org/apps/guides/hosting/ec2-setup-guide/), so I don't really know it in detail.  I can add a bit to the "why" like so (and I think delete that "note", I think it's to ourselves and not needed):

>    - DO NOT REMOVE `/srv/storage/medic-core/`, `/srv/settings/medic-core/couchdb/local.ini`. `medic-core` directory is where the CHT stores user data and `local.ini` store configuration changes, both of which we don't want to delete.  Otherwise, wipe all other files in `/srv` to prepare for the upgrade.

And then can you check about the couchdb configs you mentioned?

Reply:
The other location where customized settings could sit is the `local.d` directory. Meaning `/srv/settings/medic-core/couchdb/local.d` shouldn't be deleted as well.

Reply:
Great - thanks! I've added this in eb7055c